Marlin/configuration_store.cpp

```diff
@@ -1229,8 +1284,11 @@ void Config_ResetDefault() {
         SERIAL_ECHOLNPGM("Unified Bed Leveling:");
         CONFIG_ECHO_START;
       }
-
-      SERIAL_ECHOLNPAIR("  M420 S", ubl.state.active ? 1 : 0);
+      SERIAL_ECHOPAIR("  M420 S", ubl.state.active ? 1 : 0);
+      //#if ENABLED(ENABLE_LEVELING_FADE_HEIGHT)
+      //  SERIAL_ECHOLNPAIR(" Z", ubl.state.g29_correction_fade_height);
+      //#endif
```

Comment:
`M420 Z` is used to set Z fade height, so if you want you can remove the `F` parameter from UBL's `G29` and implement `M420 Z` for UBL instead.